python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -14,10 +14,10 @@ def start_ganga():
     file_path = os.path.dirname(os.path.realpath(__file__))
     ganga_python_dir = os.path.join(file_path, '..', '..', '..')
     ganga_python_dir = os.path.realpath(ganga_python_dir)
-    sys.path.insert(0, ganga_python_dir)
+    if len(sys.path) >= 1 and ganga_python_dir != sys.path[0]:
+        sys.path.insert(0, ganga_python_dir)
```

Comment:
This is to avoid adding it multiple times.
We don't 'need' it to be at the start, but by convention it has been for a long time (at least on LHCb) and changing it would be something I want to be careful with.